test_regress/t/t_vthread_selftest.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env perl
```

Comment:
Please rename to t_a3_selftest_thread, so sorts next to similar test.  You can use top_module("t/t_a3_selftest.v") and then remove needing a new .v file.